tripal/tripal_views_query.inc

```diff
@@ -132,6 +132,42 @@ class tripal_views_query extends views_plugin_query {
       $this->cquery->fieldCondition($field_name, $element_name, $value, $operator);
     }
   }
+  
+  /**
+   * Add's a where exression clause to a query.
```

Comment:
expression.  but i cant keep pretending typos in the comments is the point of code review lol.